### ISSUE TITLE
vlogin.php

### ISSUE THREAD

**Issue by xC4r** [xC4r/login#1]
<!DOCTYPE html>  
<html lang="es">
   <head>
      <meta charset="utf-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
      <meta name="description" content="">
      <meta name="author" content="">
      <title>Login</title>
      <link   rel="icon" href="https://getbootstrap.com/favicon.ico">
      <link   rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
      <link   rel="stylesheet" href="signin.css">
      <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
      <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
   </head>
   <body class="text-center">
      <form class="form-signin">
         <img class="mb-4" src="https://getbootstrap.com/docs/4.1/assets/brand/bootstrap-solid.svg" alt="" width="72" height="72">
         <h1 class="h3 mb-3 font-weight-normal">Login</h1>
         <label for="user" class="sr-only">Username</label>
         <input type="text" id="user" class="form-control" placeholder="Username" required autofocus>
         <label for="pass" class="sr-only">Password</label>
         <input type="password" id="pass" class="form-control" placeholder="Password" required>
         <div class="checkbox mb-3">
           <label>
             <input type="checkbox" value="remember-me"> Recordar
           </label>
         </div>
         <button class="btn btn-lg btn-primary btn-block" type="submit">Ingresar</button>
         <div id='msg'> </div>
         <p class="mt-5 mb-3 text-muted">© 2017-2018</p>
      </form>
      <script src="jlogin.js"></script>
   </body>

</html>
